gnss/gps_time.py

```diff
@@ -162,14 +163,15 @@ def utc_to_gpst(utc):
       A dictionary with attributes 'wn' and 'tow' holding the
       week number and time of week that correspond to the input utc times.
 
-    See also: datetime_to_tow, gpst_to_utc
+    See also: gpst_to_utc
     """
     # GPS-UTC offset is defined in GPS time, so some iteration is needed
+
     delta_utc_tmp = gps_minus_utc_seconds(utc)
     delta_utc_tmp = np.array(delta_utc_tmp)
+    utc = pd.to_datetime(utc)
```

Comment:
Moving this here allows this function to be passed `datetime.datetime` (as the docstring claims is allowed).